docs/application/vscode-ext/web.md

```diff
@@ -406,6 +377,93 @@ VS Code Extension for Tizen Web supports intelliSense. You can use these feature
 
   ![Document Viewer](media/tizen_web23.gif)
 
+## Tizen Web with Design Editor
+
+Visual Studio Code Extension for Tizen Web that supports WYSIWYG Design Editor features such as **Preview**, **JavaScript Assistant**, **Structure View**, **New Page**, **HTML Assistant** , **Undo** and **Redo**.
+You must select the **HTML extension file** in the file explorer before you launch Design Editor.
+
+### Run Design Editor
+
+To run Design Editor, perform one of the following:
+
+- Using Command Palette
+    1. In VS Code, click **View** > **Command Palette**.
+    2. In the input field of the **Command Palette** that appears, type **Tizen Web: Run Design-Editor** and press **Enter**.
+
+        A new tab is created and the Design Editor is launched.
+
+- Using Design-Editor button
+
+    To launch the Design Editor, click **Design-Editor** on the VS Code window as shown in the following figure:
+
+    ![Design Editor button](media/tizen_web27.png)
+
+- Using right-click options
+    1. Select the HTML file that you want to launch and right-click.
+    2. From the list that appears, click **Tizen Web: Run Design-Editor**.
+
+        ![Design Editor right click](media/tizen_web28.png)
+
+### Features
+
+Following are the WYSIWYG Design Editor features:
+
+- Drag & Drop a component
```

Comment:
From the previous comment, https://github.com/Samsung/tizen-docs/pull/1016#discussion_r375628280, only the article issue is fixed. I have also asked to change "&" to "and".
Is there a specific reason to keep "&"?

Reply:
done